src/days-2-4-adv/adv.py

```diff
@@ -22,30 +24,127 @@
 }
 
 
-# Link rooms together
+# Link rooms together -- I did not use these...
+
+# room['outside'].n_to = room['foyer']
+# room['foyer'].s_to = room['outside']
+# room['foyer'].n_to = room['overlook']
+# room['foyer'].e_to = room['narrow']
+# room['overlook'].s_to = room['foyer']
+# room['narrow'].w_to = room['foyer']
+# room['narrow'].n_to = room['treasure']
+# room['treasure'].s_to = room['narrow']
 
-room['outside'].n_to = room['foyer']
-room['foyer'].s_to = room['outside']
-room['foyer'].n_to = room['overlook']
-room['foyer'].e_to = room['narrow']
-room['overlook'].s_to = room['foyer']
-room['narrow'].w_to = room['foyer']
-room['narrow'].n_to = room['treasure']
-room['treasure'].s_to = room['narrow']
 
 #
 # Main
 #
 
-# Make a new player object that is currently in the 'outside' room.
+# Make a new player object that is currently in the 'outside' room. DONE
+
+# p1 = Player('Brian', room['outside'].room_name, room['outside'].room_description)
+
+p_name = input("Player Name: ")
+
+p1 = Player(p_name)
 
 # Write a loop that:
 #
-# * Prints the current room name
-# * Prints the current description (the textwrap module might be useful here).
-# * Waits for user input and decides what to do.
-#
-# If the user enters a cardinal direction, attempt to move to the room there.
-# Print an error message if the movement isn't allowed.
-#
-# If the user enters "q", quit the game.
+# * Prints the current room name DONE
+# * Prints the current description (the textwrap module might be useful here). DONE
+# * Waits for user input and decides what to do. DONE
+
+
+def set_player_direction(player):
+
+    print("___PLAYER_ROOM___: " + player.room_name)
+    print("___PLAYER_ROOM DESCRIPTION___: " + player.room_description)
+
+    setting_direction = [True]
```

Comment:
perhaps you intended to use Boolean True? which doesn't include brackets

Reply:
Also, If I'm thinking about this correctly, setting_direction is meant to determine whether the while loop will run. But here it looks like setting_direction is never set to False, so is it necessary?